add new event and speakers for Vue.js Israel meetup on 27 Oct 2025

AlexNechayev opened the pull request:
- Updated event details for "Vue Patterns" including metadata, agenda, and sponsors.
- Added new speaker: Shira Gold (Frontend Architect at Axonius) with speaker details.
- Introduced new talks: "What good is an app without happy users?" and "Vue Meets TanStack DB Live Queries, Optimistic Updates, and Beyond" related to the event, enhancing the content for attendees.

dimshik100 commented on content/talks.json:
```diff
@@ -493,5 +493,35 @@
+      "src": "/speaker-posters/Vue.js-Israel-speakers-adir-kandel-27.10"
+    },
+    "videos": [""],
+    "slides": {
+      "title": "",
+      "link": ""
```
We have a bug on the talk cards, when clicking the slides icon that has no link it redirects to a page with other speakers. 
The expected behaviour should be not presenting the icon if there are no slides value.

dimshik100 commented on content/speakers.json:
```diff
@@ -181,5 +181,12 @@
+    "name": "Shira Gold",
+    "title": "Frontend Architect",
+    "image": "/speaker-pictures/Shira Gold.jpeg",
+    "socialLinks": [],
```
Please add Shiras social links. at least linkedin.

dimshik100 commented on content/events.json:
```diff
@@ -1431,5 +1431,109 @@
+      "videos": {
+        "sectionTitle": "📹 The full playlist",
+        "sectionContent": {
+          "value": [""]
```
There are videos from axonius here
https://www.youtube.com/playlist?list=PLDOkyp-lEYFimI7vnCskLxPC08yodgHCD

dimshik100 replied:
If you are already handling this, please use this link to extract the "full playlists of each event" 
https://www.youtube.com/@vuejsisrael/playlists

Click on the "view full playlist" little links under each one of them to get the URL of the full list.

dimshik100 commented on content/talks.json:
```diff
@@ -493,5 +493,35 @@
+    "talkPoster": {
+      "src": "/speaker-posters/Vue.js-Israel-speakers-adir-kandel-27.10"
+    },
+    "videos": [""],
```
Please add video link

dimshik100 commented on content/talks.json:
```diff
@@ -493,5 +493,35 @@
+    "talkPoster": {
+      "src": "/speaker-posters/Vue.js-Israel-speakers-shira-gold-27.10"
+    },
+    "videos": [""],
```
Please add video link

AlexNechayev committed:
feat: add new event and talks for Vue.js Israel meetup on Oct 27, 2025

- Introduced event details including metadata, agenda, venue, and sponsors.
- Added two talks with descriptions, speaker information, and associated event IDs.
- Updated speaker information for Shira Gold and added a new speaker entry.
- bugfix - when clicking the slides icon that has no link it redirects to a page with other speakers
